src/lib/utils/hooks/SynapseAPI/team/useTeamMembers.ts

```diff
@@ -31,3 +33,13 @@ export function useGetIsUserMemberOfTeam(
     options,
   )
 }
+
+export function useIsCurrentUserACTMember() {
+  const { data: currentProfile } = useGetCurrentUserProfile()
+  const currentUserId = currentProfile?.ownerId
+    ? currentProfile.ownerId.toString()
+    : ''
+  return useGetIsUserMemberOfTeam(ACT_TEAM_ID.toString(), currentUserId, {
```

Comment:
Reason why you are checking team membership instead of asking for the user bundle (for is ACT member)?  https://rest-docs.synapse.org/rest/GET/user/bundle.html

Reply:
Good catch, that would be more appropriate